rxbinding-leanback-v17/src/main/java/com/jakewharton/rxbinding/support/v17/leanback/widget/RxSearchBar.java

```diff
@@ -0,0 +1,62 @@
+package com.jakewharton.rxbinding.support.v17.leanback.widget;
+
+import android.support.annotation.CheckResult;
+import android.support.annotation.NonNull;
+import android.support.v17.leanback.widget.SearchBar;
+import rx.Observable;
+import rx.functions.Action1;
+
+/**
+ * Static factory methods for creating {@linkplain Observable observables} and {@linkplain Action1
+ * actions} for {@link SearchBar}.
+ */
+public final class RxSearchBar {
+  /**
+   * Create an observable of {@linkplain SearchBarSearchQueryEvent search query events} on {@code
+   * view}.
+   * <p>
+   * <em>Warning:</em> The created observable keeps a strong reference to {@code view}. Unsubscribe
+   * to free this reference.
+   * <p>
+   * <em>Note:</em> A value will not be emitted on subscribe.
```

Comment:
I don't think we put this "not" warnings anywhere. Only when there will be an initial value do we add the warning.